test/include/userobjects/InterfaceUserObjectTestGetMaterialProperty.h

```diff
@@ -0,0 +1,44 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InterfaceUserObject.h"
+
+class InterfaceUserObjectTestGetMaterialProperty;
+
+template <>
+InputParameters validParams<InterfaceUserObjectTestGetMaterialProperty>();
```

Comment:
legacy

Reply:
as before

Reply:
Do we still need this one? I don't consider these a roadblock to merging; I only ask because I thought you indicated you were able to rip all the legacy out